felayout_t3kit/dev/styles/main/contentElements/logoCarousel.less

```diff
@@ -7,10 +7,6 @@
     padding: 0 30px;
 }
 
-.logo-carousel__slide {
-    padding: 10px 30px;
```

Comment:
I do not agree with this fix. If you remove padding from class .logo-carousel__slide it will be removed not only from the image without a link but also from the image with a link which causes problems in logo carousel element (logos will be too big)

Reply:
`padding: 10px 30px;`
![screen shot 2018-04-24 at 2 09 19 pm](https://user-images.githubusercontent.com/5150636/39183596-4027bbc8-47c9-11e8-9c4e-6c7f01c8cd80.png)

`-    padding: 10px 30px;`
![screen shot 2018-04-24 at 2 08 49 pm](https://user-images.githubusercontent.com/5150636/39183587-3a118ba6-47c9-11e8-984a-1064f0d0be7c.png)

Reply:
Here is a Screenshot of what I mean: 
![image](https://user-images.githubusercontent.com/31850727/39187749-61a11772-47ce-11e8-9192-978849878364.png)

The Problem is that there is some css styling on the `.logo-carousel__link` class which means when there is no link set there is some styling missing. 
The only visible difference is the padding.

I can either make the icons all the smaller size or the larger size, which one would you like?

Reply:
My main concerns are that this fix can cause changes in existing websites after the update as I showed it on screenshots above. So you need to add changes without changing existed behavior. I would suggest making not linked files smaller.